src/App/tests/App.test.tsx

```diff
@@ -62,7 +62,7 @@ describe('App', () => {
       </Provider>
     );
     // before resolving get oauth state request, the user is logged out
-    expect(wrapper.text()).toMatchInlineSnapshot(`"HomeLogin"`);
+    expect(wrapper.text()).toMatchInlineSnapshot(`"HomeLoginreveal-frontend: 0.4.2-rc7"`);
```

Comment:
Do you think it is worthwhile to have an additional snapshot test just for the footer/other sections?  I find the current test to be a bit strange i.e. this test just shows that this text appears on this page, right?

Reply:
yeah, sure